### PR TITLE
Pass unknown commands to yotta

### LINKED ISSUE #8: Enable yotta command passthrough
Currently any commands which are not directly supported by kubos-sdk are spit back, they should be passed along to yotta.

### PR DESCRIPTION
**Pull request by kyleparrott**
This works by:

- Completely overriding the error handler in the host script (Allows passing the command to the container)
- Temporarily overriding the same handler in the container script
 - Before the command is passed to yotta the handler is set back to its default

This wil fix #8

### CODE REVIEW

File: kubos-sdk.py [hunk 1]
```diff
@@ -32,6 +32,20 @@
 from yotta import build, init, target
 from yotta.lib import component, globalconf
 
+# Temporarily override the argparse error handler that deals with undefined subcommands
+# to allow these subcommands to be passed to yotta. Before calling yotta
+# the error handler is set back to the standard argparse handler.
+
+original_check_value = argparse.ArgumentParser._check_value
+
+def kubos_check_value(self, action, value):
```
Comment (rplauche) [inline kubos-sdk.py]: Can we move this function out of the variable initializations and down with the other functions?

File: kubos [hunk 2]
```diff
@@ -25,6 +25,17 @@ import time
 from docker import Client
 import docker
 
+# Override the argparse error handler that deals with undefined subcommands
+# to allow these commands to be passed through to the container. The container
+# handles passing commands to yotta as well as subcommands unknown to yotta.
+
+def pass_through_check_value(self, action, value):
```
Comment (rplauche) [inline kubos]: Let's move this function down with the other functions

**Comment by kyleparrott** [comment]
@rplauche: updated

File: kubos-sdk.py (hunk 1 above)
Comment (rplauche) [inline kubos-sdk.py]: These variable assignments (original_check_value and argparse.ArgumentParser._check_value) should done at the top with the other variables, not mixed in with the function declarations.

File: kubos (hunk 2 above)
Comment (rplauche) [inline kubos]: Same comment about organization of variable assignment and function declaration.

File: kubos-sdk.py (hunk 1 above)
Comment (kyleparrott) [inline kubos-sdk.py]: My bad for missing this and moving it all. But doesn't line 158, 
`argparse.ArgumentParser._check_value = kubos_check_value` 
need to stay after the kubos_check_value function definition?

**Comment by rplauche** [comment]
Looks good to merge.

**Pull request merged**